api/candlepin-api-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "modelPackage": "org.candlepin.dto.api.v1",
```

Comment:
What are the implications of the package changing with API version differences? That is, how painful is it going to be if we change this to a v2 or some such in the future?

Reply:
We could always have two API specification files that generate to different package names.  That wouldn't be awesome, but that's one way we could manage a change to v2.